### PR TITLE
fix mlflow + push to gcp

### PR DESCRIPTION
**Pull request by barangerbenjamin**
**data.py**

_Training on all data now_

**training.py**

_Fix mlflow and remove GPU_

**.gitignore**

_Add Fake_news/raw_data_

**Makefile**

_Just modified the job name_

### CODE REVIEW

File: Fake_news/data.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # from Fake_news.params import #
 
 
-def get_data(local=False, sample_size=0.0005):
+def get_data(local=False, sample_size=1):
```
Comment (barangerbenjamin) [inline Fake_news/data.py:6]: Training on all data

File: Fake_news/trainer.py [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 import tensorflow as tf
 import numpy as np
+import mlflow
+from  mlflow.tracking import MlflowClient
```
Comment (barangerbenjamin) [inline Fake_news/trainer.py:6]: Add missing import for MLFlow

File: Fake_news/trainer.py [hunk 3]
```diff
@@ -122,13 +124,13 @@ def init_model():
...
-        strategy = tf.distribute.MirroredStrategy()
+        # strategy = tf.distribute.MirroredStrategy()
 
         # Open a strategy scope.
-        with strategy.scope():
+        # with strategy.scope():
             # Everything that creates variables should be under the strategy scope.
             # In general this is only model construction & `compile()`.
-            self.model = init_model()
+        self.model = init_model()
```
Comment (barangerbenjamin) [inline Fake_news/trainer.py:133]: Remove GPU (that has somehow reduced job time by 3)

File: Fake_news/trainer.py [hunk 4]
```diff
@@ -174,7 +176,7 @@ def mlflow_run(self):
         return self.mlflow_client.create_run(self.mlflow_experiment_id)
 
     def mlflow_log_param(self, key, value):
-        self.mlflow_log_metriclient.log_param(self.mlflow_run.info.run_id, key, value)
+        self.mlflow_client.log_param(self.mlflow_run.info.run_id, key, value)
```
Comment (barangerbenjamin) [inline Fake_news/trainer.py:179]: typo to log on MLFlow

File: Makefile [hunk 5]
```diff
@@ -70,7 +70,7 @@ PYTHON_VERSION=3.7
 PACKAGE_NAME=Fake_news
 FILENAME=main
 RUNTIME_VERSION=2.1
-JOB_NAME=fake_news_training_pipeline_$(shell date +'%Y%m%d_%H%M%S')
+JOB_NAME=fake_news_training_pipeline_no_gpu_$(shell date +'%Y%m%d_%H%M%S')
```
Comment (barangerbenjamin) [inline Makefile:73]: Change job name to be easily identified

**Review by DataASG — APPROVED** [review]

**Pull request merged**